conversionF1.c

```diff
@@ -0,0 +1,16 @@
+bool characteristic(char numString[], int& c);
+bool mantissa(char numString[], int& numerator, int& denominator);
+...
+...
```

Comment:
Omit this (we know to put more code in here)

Reply:
Will do. Thanks for the input.